sputnik-staking/src/lib.rs

```diff
@@ -0,0 +1,217 @@
+use near_contract_standards::fungible_token::receiver::FungibleTokenReceiver;
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::collections::LookupMap;
+use near_sdk::json_types::{ValidAccountId, WrappedDuration, U128};
+use near_sdk::{
+    env, ext_contract, near_bindgen, AccountId, Balance, BorshStorageKey, Duration, Gas,
+    PanicOnDefault, Promise, PromiseOrValue, PromiseResult,
+};
+
+pub use user::{User, VersionedUser};
+
+mod storage_impl;
+mod user;
+
+#[derive(BorshStorageKey, BorshSerialize)]
+enum StorageKeys {
+    Users,
+}
+
+/// Amount of gas for fungible token transfers.
+pub const GAS_FOR_FT_TRANSFER: Gas = 10_000_000_000_000;
+
+/// Amount of gas for delegate action.
+pub const GAS_FOR_DELEGATE: Gas = 10_000_000_000_000;
+
+/// Amount of gas for register action.
+pub const GAS_FOR_REGISTER: Gas = 10_000_000_000_000;
+
+/// Amount of gas for undelegate action.
+pub const GAS_FOR_UNDELEGATE: Gas = 10_000_000_000_000;
+
+/// TODO: this should be in the near_standard_contracts
+#[ext_contract(ext_fungible_token)]
+pub trait FungibleToken {
+    fn ft_transfer(&mut self, receiver_id: AccountId, amount: U128, memo: Option<String>);
+}
```

Comment:
It is `use near_contract_standards::fungible_token::core_impl::ext_fungible_token;`